### PR TITLE
feat: Create the Player Stats App

### PR DESCRIPTION
**Pull request by gvgesanto2**
## Description

**Task:** https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md

This PR makes the following changes:
-  Creates the Player Stats App using the [Football API](https://www.api-football.com/)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project (no linters errors)
- [X] I have checked my code and corrected any misspellings
- [X] I have deleted any unused code as comments in my codebase

## Screenshots

- **``Homepage`` on Desktop version:**
image
- **``Homepage`` on Tablet version:**
image
- **``Homepage`` on Mobile version:**
image

- **``PlayerPropfilePage`` in the Desktop version:**
image
image
- **``PlayerPropfilePage`` on Table version:**
image
- **``PlayerPropfilePage`` on Mobile version:**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+  - React
+  - Redux & Redux Thunk
+
+- **Conventions and Methodologies:**
+  - BEM
+
+## Documentation Video
+
+Click on this link to see the documentation video: [Doc Video Link](https://www.loom.com/share/c703ee2a86864583a0fbbe748e4f57a7)
+
```
Comment (ShahierNashaat) [inline README.md:27]: - [ ] Kindly be noticed that the video should contain a highlight about a piece of the code you finds interesting or is proud of making.

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+- **Languages & Technologies:**
+  - HTML5 & CSS3
+  - Sass
+  - JavaScript
+  - React
+  - Redux & Redux Thunk
+
+- **Conventions and Methodologies:**
+  - BEM
+
```
Comment (ShahierNashaat) [inline README.md:19]: - [ ] Kindly be noticed that you should deploy the application using Heroku, Netlify, or GitHub Pages.
- [ ] Also the link to the deployed project is present in the README.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @gvgesanto2, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- All tests are passing ✔️
- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @gvgesanto2, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**